content/company/culture.mdx

```diff
@@ -3,9 +3,8 @@ seo:
   title: SSW Culture
   description: >
     SSW's company culture is rooted in .NET consultancy, Scrum methodologies, 
-    and software development best practices. Experience our "Make it happen" 
-    approach, commitment to workplace happiness, and innovative tech solutions 
-    in Australia and beyond.
+    and software development best practices. Our "Make it happen" makes us one of the most effective, efficient, and 
```

Comment:
Should this say `"Make it happen" approach`?

Reply:
yes, test pass otherwise